Formula/emqx.rb

```diff
@@ -0,0 +1,69 @@
+class Emqx < Formula
+  desc "MQTT broker for IoT"
+  homepage "https://www.emqx.io/"
+  url "https://github.com/emqx/emqx/archive/refs/tags/v5.0.9.tar.gz"
+  sha256 "bc5d193a02840b1d4cb5f6c9f047003b442c5a0b4a3ccb1cfeedafacf04ff82e"
+  license "Apache-2.0"
+  head "https://github.com/emqx/emqx.git", branch: "master"
+
+  def dashboard_version
+    "1.1.0"
+  end
+
+  def emqx_version
+    "5.0.9"
+  end
+
+  depends_on "autoconf"  => :build
+  depends_on "automake"  => :build
+  depends_on "cmake"     => :build
+  depends_on "coreutils" => :build
+  depends_on "erlang@24" => :build
+  depends_on "freetds"   => :build
+  depends_on "libtool"   => :build
+  depends_on "openssl@1.1"
+
+  uses_from_macos "curl"  => :build
+  uses_from_macos "unzip" => :build
+  uses_from_macos "zip"   => :build
+
+  def install
+    ENV["PKG_VSN"] = emqx_version
+    ENV["PROFILE"] = "emqx"
+    ENV["EMQX_DASHBOARD_VERSION"] = "v#{dashboard_version}"
+    system "make", "ensure-rebar3"
+    system "./build", "emqx", "rel"
+    mkdir(prefix)
+    system "rsync", "-a",
+           "--exclude", "'*.tar.gz'",
+           "--exclude", "no_dot_erlang.boot",
+           "--exclude", "emqx_ctl.cmd",
+           "--exclude", "emqx.cmd",
+           "_build/emqx/rel/emqx/", "#{prefix}/"
+  end
+
+  def caveats
+    <<~EOS
+      Start/stop emqx daemon:
+
+         #{HOMEBREW_PREFIX}/bin/emqx start
+         #{HOMEBREW_PREFIX}/bin/emqx stop
+
+      To fix ulimit if you see the warning (recommended):
+
+         launchctl limit maxfiles 2048 2048
+
+      More commands:
+
+         #{HOMEBREW_PREFIX}/bin/emqx help
+
+    EOS
+  end
+
+  test do
+    exec "ln", "-s", testpath, "data"
+    ENV["EMQX_WAIT_FOR_START"] = "5"
+    exec bin/"emqx", "start"
```

Comment:
Can you make sure this works?

Reply:
Yes, if it won't start in 5 seconds, the script will terminate with error exit code.

Reply:
What I mean is that this script has auto-verification of successful start built in https://github.com/emqx/emqx/blob/master/bin/emqx#L827

Reply:
It might start succesfully, but can you also add a verification that it does what it's supposed to after?

Reply:
It does many things, I cannot test all of them. Successful start is a good indicator that the application has been installed successfully.